eventing/sources/sources.yaml

```diff
@@ -59,6 +59,13 @@ sources:
     description: >
       Registers for events of the specified types on the specified GitLab repository. Brings those
       events into Knative.
+  - name: BitBucket
```

Comment:
Follow the instructions in event/sources/README.md, namely

2. Run the generator tool:
    ```shell
    go run eventing/sources/generator/main.go
    ```

Reply:
Thanks @Harwayne ! Forgot to run it.